Q/QuantRegBuilder/build_tarballs.jl

```diff
@@ -0,0 +1,41 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "QuantRegBuilder"
```

Comment:
Sorry, I just realised that this should be called
```suggestion
name = "QuantReg"
```
Please, rename also the directory to exactly match the new name, and then we're ready to go!